doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+Using Ubuntu as the Service OS with the ACRN hypervisor
+
+
+Based on the ACRN open source project and getting started guide. This documentation is to guide people how to boot Ubuntu system as the 
+Service OS with the ACRN hypervisor. By default, in the getting started guide documentation of the ACRN open source project, Clear Linux
+is being used as the Service OS and User OS on top of the ACRN hypervisor. 
+Here we are trying to boot Ubuntu system as the Service OS and launch Clear Linux as User OS with the ACRN hypervisor on official 
+supported NUC, playing steps as below:
+
+Step #1,  Installing a Ubuntu system on the NUC with standard installing steps. 
```

Comment:
I believe it's still possible to install Ubuntu on a standard Bios system. IIRC, the boot mechanism that will be used is the same as the one that was used to boot of the installation medium. If this is true, we should add a note telling folks that they **must** install using UEFI.

Reply:
We should also recommend that people do **not** have `secure boot` on.